styles/base/index.scss

```diff
@@ -1,2 +1,13 @@
 @use 'reset';
 @use 'elements';
+
+* {
+  box-sizing: border-box;
+}
+
+button,
+html input[type='button'],
+input[type='reset'],
+input[type='submit'] {
+  cursor: pointer;
+}
```

Comment:
Should these rules not go in `_elements.scss`?

Reply:
My plan is to entirely obliterate `_elements` in a few steps from now, so this is a place to isolate remaining things that I need to take a second look at.